### PR TITLE
Add support for non-default version of ES

### PR DESCRIPTION
**Pull request by IlyaSukhanov**

### CODE REVIEW

**Comment by IlyaSukhanov** [comment]
build

File: disco_aws_automation/disco_elasticsearch.py [hunk 1]
```diff
@@ -393,6 +393,7 @@ def _get_es_config(self, elasticsearch_name):
         allowed_source_ips = self.get_es_option("allowed_source_ips", elasticsearch_name).split()
 
         config = {
+            'ElasticsearchVersion': self.get_es_option('version', elasticsearch_name),
```
Comment (jconstance-amplify) [inline disco_aws_automation/disco_elasticsearch.py]: Worthwhile to add any unit tests asserting this is picked up/sent correctly?

**Comment by jichen-amplify** [comment]
👍

**Comment by pchand-amplify** [comment]
👍

**Comment by aoskotsky-amplify** [comment]
👍

File: disco_aws_automation/disco_elasticsearch.py (hunk 1 above)
Comment (IlyaSukhanov) [inline disco_aws_automation/disco_elasticsearch.py]: I have mixed feeling about this, if we were to check value change in mock model that's one thing. Else were just testing that we pass to the interface what we expect to pass.

Comment (jconstance-amplify) [inline disco_aws_automation/disco_elasticsearch.py]: Fair enough.

**Pull request merged**